src/main/java/com/google/sps/servlets/GoogleAuthServlet.java

```diff
@@ -0,0 +1,156 @@
+
+    private void storeUserCredentials(GoogleTokenResponse tokenResponse,  String userId){
+
+        JsonObject userCredentials = new JsonObject();
+        userCredentials.addProperty("id", userId);
+        userCredentials.addProperty("access_token", tokenResponse.getAccessToken());
+        userCredentials.addProperty("refresh_token", tokenResponse.getRefreshToken());
+        userCredentials.addProperty("scope", tokenResponse.getScope());
+        userCredentials.addProperty("id_token", tokenResponse.getIdToken());
+        userCredentials.addProperty("token_type", tokenResponse.getTokenType());
+        userCredentials.addProperty("expires_in", tokenResponse.getExpiresInSeconds());
+
+        DatastoreModule.storeUserCredentials(userCredentials, userId);
+
+
+
+        /**
+         *  .set("timestamp", timeStampInSeconds)
+                .set("id", profileId)
+                .set("access_token", userCredentials.get("access_token").toString().replaceAll(quotes,""))
+                .set("refresh_token", userCredentials.get("refresh_token").toString().replaceAll(quotes,""))
+                .set("scope", userCredentials.get("scope").toString().replaceAll(quotes,""))
+                .set("token_type", userCredentials.get("token_type").toString().replaceAll(quotes,""))
+                .set("expires_in", userCredentials.get("expires_in").toString().replaceAll(quotes,""))
+                .set("id_token", userCredentials.get("id_token").toString().replaceAll(quotes,""))
+                .build();
+         */
+
+    }
+    
+    private void storeUserInfo(GoogleIdToken.Payload payload, String userId){
+        
+        JsonObject userInfo = new JsonObject();
+        userInfo.addProperty("id", userId);
+        userInfo.addProperty("name", (String) payload.get("name"));
+        userInfo.addProperty("given_name", (String) payload.get("given_name"));
+        userInfo.addProperty("family_name", (String) payload.get("family_name"));
+        userInfo.addProperty("picture", (String) payload.get("picture"));
+        userInfo.addProperty("locale", (String) payload.get("locale"));
+        userInfo.addProperty("email", payload.getEmail());
+
+        DatastoreModule.storeUserInfo(userInfo);
+        /**
+         * Entity userEntity = Entity.newBuilder(keyFactory.newKey(userInfo.get("id").toString().replaceAll(quotes,"")))
+                                      .set("id", userInfo.get("id").toString().replaceAll(quotes,""))
+                                      .set("name", userInfo.get("name").toString().replaceAll(quotes,""))
+                                      .set("given_name", userInfo.get("given_name").toString().replaceAll(quotes,""))
+                                      .set("family_name", userInfo.get("family_name").toString().replaceAll(quotes,""))
+                                      .set("picture", userInfo.get("picture").toString().replaceAll(quotes,""))
+                                      .build();
+         */
+    }
```

Comment:
Remove commented out code. You can move these off to other branches in order to cherry pick them in when you need to test things but never commit them into main branch.